modules/learn/pages/buckets-memory-and-storage/storage-engines.adoc

```diff
@@ -1,95 +1,119 @@
 = Storage Engines
-:description: pass:q[Couchbase supports two different backend storage mechanisms: Couchstore and Magma.]
+:description: pass:q[Couchbase supports two different backend storage engines: Couchstore and Magma.]
 
 [abstract]
 {description}
-It is important to understand which backend storage is best suited to your requirements.
+These storage engines organize the data both on disk ad in memory.
+This page explains how to choose a  backend storage to suit your needs.
 
-== Couchstore
-
-Couchstore is the default bucket storage engine.
-It is optimized for high performance with large data sets,
-while using fewer system resources (the minimum bucket size for the Couchstore backend is 100{nbsp}MB).
-If you have a small data set that can fit in memory, then you should consider using Couchstore.
 
 [#storage-engine-magma]
 == Magma
 
 [.edition]#{enterprise}#
 
-Magma is designed for high performance with very large datasets that do not fit in memory.
-It is ideal for use cases that rely primarily on disk access.
-The performance of disk access will be as good as the underlying disk sub-systems --
-for example, using NVMe SSDs will give higher performance.
+Magma is designed for high performance with large datasets that do not fit in memory.
+It's ideal for use cases that rely primarily on data stored on  disk.
+The performance of disk access is as good as the underlying disk sub-systems.
+For example, using NVMe SSDs gives better performance than spinning drives.
 
-In order to get maximum performance from Magma for disk-oriented workloads, it is recommended to set the Writer Threads to `Disk i/o optimized`. This setting will ensure there are enough threads to sustain high write rates.
+When you create a bucket using the Magma storage engine, you choose the number of xref:learn:buckets-memory-and-storage/vbuckets.adoc[vBuckets] a bucket uses.
+Magma supports two vBucket configurations: 128 or 1024.
+This choice affects the minimum memory quota for the bucket.
+The 128 vBucket configuration is the default for new buckets in Couchbase Server Enterprise Edition 8.0 and later.
+It has a minimum memory quota of 100{nbsp}MiB per node.
 
-To learn more about Writer Thread settings, see xref:manage:manage-settings/general-settings.adoc#data-settings[Data Settings]
+NOTE: Magma buckets with 128 vBuckets are only available in clusters that have fully migrated to Couchbase Server 8.0 or later.
 
-Magma can work with very low amounts of memory for large datasets: a minimum memory-to-data ratio of 1% is required.
-For example, if a node is holding 5{nbsp}TB of data, Magma can be used with only 64{nbsp}GB RAM.
+Magma using 1024 vBuckets has a minimum memory quota of 1{nbsp}GiB per node.
```

Comment:
I liked the "Which Storage Engine Should You Use?", but since that section is further down in the page, I'm thinking that this may be a good place to just note here:

If you can allocate at least 1 GiB memory per node to your bucket, you should choose the 1024 vBucket option for Magma as you will get better performance at scale.